### PR TITLE
Apps - Noalyss - Fixed broken link and removed misleading information

### PR DESCRIPTION
**Pull request by oleole39**
In Noalyss app docs:
- Fixed link to user documentation leading to a 404 page
- Removed misleading login information which is likely to be related to an old version of the package 

You might find it difficult to check the relevancy of the second point since current YNH package for this app is broken. This pull request is meant to be associated to [that other pull request](https://github.com/YunoHost-Apps/noalyss_ynh/pull/39) which aims at fixing the current package.

### CODE REVIEW

**Review by tituspijean — APPROVED** [review]

**Comment by tituspijean** [comment]
@oleole39 Can you rebase/resync your fork? You are 707 commits behind the master branch here. ;)

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Review by ericgaspar — APPROVED** [review]

File: pages/04.applications/10.docs/noalyss/app_noalyss.fr.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ routes:
...
 Logiciel de comptabilité open source développé en PHP.
 
-Toute les infos sur ce logiciel sont disponible sur le [site du projet](http://www.noalyss.eu)
+Toute les infos sur ce logiciel sont disponible sur le [site du projet](http://www.noalyss.eu).
 
-Pour apprendre à l'utiliser je vous invite à lire le [wiki du projet](http://www.noalyss.eu/?page_id=46&lang=fr_FR)
+Pour apprendre à l'utiliser je vous invite à lire la [documentation utilisateur du projet](https://www.noalyss.eu/?page_id=1031).
 
-Le github du module YunoHost est [ici](https://github.com/YunoHost-Apps/noalyss_ynh)
+Le github du module YunoHost est [ici](https://github.com/YunoHost-Apps/noalyss_ynh).
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/noalyss/app_noalyss.fr.md:16]:
```suggestion
Le GitHub du module YunoHost est [ici](https://github.com/YunoHost-Apps/noalyss_ynh).
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: pages/04.applications/10.docs/noalyss/app_noalyss.fr.md (hunk 1 above)
Comment (tituspijean) [reply pages/04.applications/10.docs/noalyss/app_noalyss.fr.md:16]: Ooops already merged :/

Comment (tituspijean) [reply pages/04.applications/10.docs/noalyss/app_noalyss.fr.md:16]: Solved in af9d3d3336aa4331c2f6cdf53657a894d700062e

**Pull request merged**